adapters/Vungle/VungleAdapter/VungleAdNetworkExtras.h

```diff
@@ -1,34 +0,0 @@
-//
-//  VungleAdNetworkExtras.h
-//
-//  Copyright © 2016 Vungle. All rights reserved.
-//
-
-#import <Foundation/Foundation.h>
-
-#import <GoogleMobileAds/GADAdNetworkExtras.h>
-
-@interface VungleAdNetworkExtras : NSObject<GADAdNetworkExtras>
-
-/*!
- * @brief NSString with user identifier that will be passed if the ad is
- * incentivized.
- * @discussion Optional. The value passed as 'user' in the an incentivized
- * server-to-server call.
- */
-@property(nonatomic, copy) NSString *_Nullable userId;
-
-/*!
- * @brief Controls whether presented ads will start in a muted state or not.
- */
-@property(nonatomic, assign) BOOL muted;
-
-@property(nonatomic, assign) NSUInteger ordinal;
-
-@property(nonatomic, assign) NSTimeInterval flexViewAutoDismissSeconds;
```

Comment:
Hello ashinagawa. You missed these two properties line number 26 and 28. Please add the new parameters to the [VungleAdNetworkExtras.h](https://github.com/Vungle/googleads-mobile-ios-mediation/blob/e123e5fc7367b17c02d619fbc23ab126b1fd3335/adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h) file in the `Public/Headers/` directory.